src/Compilers/CSharp/Portable/Compiler/Instrumentation.cs

```diff
@@ -48,6 +50,15 @@ internal static BoundBlock InjectInstrumentation(MethodSymbol method, BoundBlock
                     BoundExpression payloadNullTest = factory.Binary(BinaryOperatorKind.ObjectEqual, boolType, factory.Field(null, payloadField), factory.Null(payloadType));
                     BoundStatement payloadIf = factory.If(payloadNullTest, createPayloadCall);
 
+                    // Methods defined with => syntax don't have a block syntax, and so won't naturally get a sequence point for the
+                    // entry of the method. It is a requirement that there be a sequence point before any executable code. A sequence point
+                    // won't be generated automatically for the synthesized if statement because the if statement is compiler generated, so
```

Comment:
Would it be easier to just wrap the body in a block so that it naturally gets a sequence point?